automated install/basic-install.sh

```diff
@@ -1977,6 +1977,17 @@ FTLcheckUpdate()
   fi
 
   if [[ ! "${ftlBranch}" == "master" ]]; then
+    #Check whether or not the binary for this FTL branch actually exists. If not, then there is no update!
+    local path
+    path="${ftlBranch}/${binary}"
+    # shellcheck disable=SC1090
+    source "${PI_HOLE_SCRIPT_DIR}/piholeCheckout.sh"
+    #check_download_exists sourced from piholeCheckout.sh
+    if ! check_download_exists "$path"; then
+      echo -e "  ${INFO}  Download for branch \"${ftlBranch}\" is not available. Nothing to update to!"
```

Comment:
Looks good, how about some wording to tell them to check for a new branch or master?